src/Adapter/Tools.php

```diff
@@ -26,7 +26,6 @@
 
 namespace PrestaShop\PrestaShop\Adapter;
 
-use PrestaShop\PrestaShop\Adapter\File\HtaccessFileGenerator;
```

Comment:
Not needed this is used in `@see` annotation so it's acceptable, although the annotation can work without it as well so it's not a blocking topic but it's not in your PR's scope and it brings no value to remove this anyway

So TLDR; do as you like 😅